vicon_estimate.py

```diff
@@ -157,22 +157,50 @@ def get_gt_velocity(filename):
 
 
 
-    sns.kdeplot(df_vel['vel'])
-    plt.axvline(np.median(df_vel['vel']),color='k',ls='--',label=f"Median: {np.round(np.median(df_vel['vel']),4)} cm/sec")
-    plt.axvline(np.mean(df_vel['vel']),color='green',label=f"Mean: {np.round(np.mean(df_vel['vel']),4)} cm/sec")
-    plt.legend()
-    plt.savefig('mean_median_speed_estimate.pdf')
-    plt.show()
+    # sns.kdeplot(df_vel['vel'])
+    # plt.axvline(np.median(df_vel['vel']),color='k',ls='--',label=f"Median: {np.round(np.median(df_vel['vel']),4)} cm/sec")
+    # plt.axvline(np.mean(df_vel['vel']),color='green',label=f"Mean: {np.round(np.mean(df_vel['vel']),4)} cm/sec")
+    # plt.legend()
+    # plt.savefig('mean_median_speed_estimate.pdf')
+    # plt.show()
 
     df_vel['sec']=np.round(df_vel.timestamp)
 
-    plt.figure(figsize=(15,3))
-    df_vel.groupby('sec')['vel'].mean().plot(label='mean')
-    df_vel.groupby('sec')['vel'].median().plot(label='median')
-    plt.ylabel('Velocity (cm/sec)')
-    plt.legend()
-    plt.ylabel('Vicon Estiated Speed')
-    plt.xlabel('No. of Frames')
-    plt.savefig('vel_vicon.pdf')
-    plt.show()
+    # plt.figure(figsize=(15,3))
+    # df_vel.groupby('sec')['vel'].mean().plot(label='mean')
+    # df_vel.groupby('sec')['vel'].median().plot(label='median')
+    # plt.ylabel('Velocity (cm/sec)')
+    # plt.legend()
+    # plt.ylabel('Vicon Estimated Speed')
+    # plt.xlabel('No. of Frames')
+    # plt.savefig('vel_vicon.pdf')
+    # plt.show()
     return df_vel.groupby('sec')['vel'].mean()
```

Comment:
Keep the plots; no need to comment them out.

Reply:
Done